pennylane/ops/op_math/decompositions/controlled_decompositions.py

```diff
@@ -589,37 +632,56 @@ def _mcx_one_worker_condition(num_control_wires, num_work_wires, **__):
 
 def _mcx_one_worker_resource(num_control_wires, work_wire_type, **__):
     if work_wire_type == "zeroed":
-        n_ccx = 2 * num_control_wires - 3
-        return {ops.Toffoli: n_ccx, ops.X: n_ccx - 3}
+        n_ccx = 2 * num_control_wires - 5
+        return {
+            ops.Toffoli: n_ccx,
+            qml.TemporaryAND: 1,
+            adjoint_resource_rep(qml.TemporaryAND): 1,
+            ops.X: n_ccx - 1,
+        }
```

Comment:
For a `"zeroed"` work wire, we can replace one pair of `Toffoli`s by a `TemporaryAND` - `adjoint(TemporaryAND)` pair.